### ISSUE TITLE
Sonarr and Radarr calendar on homepage aren't working

### ISSUE THREAD

**Issue by masterxdk** [causefx/Organizr#881]
###### Operating System: Windows Server 2016
<hr>

##### Problem Description:
Radarr and Sonarr is configured under homepage items, but the calendar aren't being updated with data from the two. 
When configuring them, the test connection button doesn't show any notification in the bottom right corner. Obs.: I see it very very very shortly. Very quickly gone again.

Was working fine in the latest version of v1

<hr>

##### Reproduction Steps:
Sonarr and Radarr is behind a nginx reverse proxy (on a Synology box)
Organizr v2 is on a Windows 2016 VM hosted in Azure
They are on two seperate networks and two seperate public IP's and domain names.
Base URL in Sonarr and Radarr is empty
URL in Organizr is configured as https://domain:8500 and https://domain:8501
Organizr is available at https://portal.domain
Token is set as the API key from the two

Tried:
To recreate API key in Sonarr = No change
Add a trailing / to the URL = No change

<hr>

#### Errors on screen?  If so paste here:
No errors in the Organizr logs or on screen when testing connection
Chrome Console Output:
![consoleoutput](https://user-images.githubusercontent.com/1771591/40277440-4b2c5a1e-5c1f-11e8-9970-87cf29b5339c.png)
Chrome Function View:
![functionerror](https://user-images.githubusercontent.com/1771591/40277441-4b495218-5c1f-11e8-9ed7-f0ba3033ebbf.png)
```

```

**Comment by masterxdk** [comment]
Fixed by CauseFX

**Comment by causefx** [comment]
This issue was caused by Guzzle HTTP not updating or using correct Cert.

**Comment by Swiftnesses** [comment]
I see to have an issue with Radaar only...

Settings have been checked, it was working a few releases ago?

<img width="758" alt="screenshot 2018-11-08 at 08 11 44" src="https://user-images.githubusercontent.com/1001877/48185770-f946cb80-e32d-11e8-82c5-91cb8db12224.png">

**Comment by krumpaul** [comment]
@Swiftnesses did you ever find a fix?  I get the same issue. I first had to allow a couple headers due to CORS: 
        add_header Access-Control-Allow-Origin "*";
        add_header Access-Control-Allow-Headers "*";

then I get the exact error below with radarr v2.


> I see to have an issue with Radaar only on the latest V2 releases...
> 
> Settings have been checked, it was working a few releases ago?
> 
> <img alt="screenshot 2018-11-08 at 08 11 44" width="758" src="https://user-images.githubusercontent.com/1001877/48185770-f946cb80-e32d-11e8-82c5-91cb8db12224.png">

**Comment by Swiftnesses** [comment]
I re-pulled the repo, now it works for me...

**Comment by causefx** [comment]
go back to settings and use the test api connection button on both sonarr and radarr.

**Comment by metroplexing** [comment]
@krumpaul did you solve this issue on the latest v2 release? I've been getting the same error for the past week and the test API connection button (under settings) is successful (sonarr and radarr).

**Comment by krumpaul** [comment]
@causefx  the api works and will show the calendar on the home page just fine for both sonarr and radarr with that json error.  It just throws the json when trying to load in the iframe for some reason.  I only had this issue with radarr, sonarr was able to load in the iframe by default.  Radarr put some additional Access-control settings that sonarr doesnt have right now so when I reverse proxy radarr AND I want radarr to load in organizr's iframe I have to add the following to my reverse proxy to get it to work (this threw a different error, it wasnt until I put this in that I got the json error and I reinstalled a couple times to get past the json error):         

add_header Access-Control-Allow-Origin "https://{ORGANIZR URL}" always;
add_header Access-Control-Allow-Headers "*" always;
